docs/general/passing_arbitrary_options.md

```diff
@@ -0,0 +1,45 @@
+## Passing Arbitrary Options To A Serializer
+
+Let's say you have a basic Post Controller:
+
+```ruby
+class PostController < ApplicationController
+  def dashboard  
+    render json: @posts
+  end
+end
+```
+
+Odds are, your serializer will look something like this:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
+end
+```
+
+This works all fine and well, but maybe you passing in some "arbitrary" options
+into the serializer. Here's what you would do:
+
+### posts_controller.rb
```

Comment:
I really don't mine that at all. Definitely could flow a bit better. Going to do an A / B comparison and see what's up.